tests/utils/fixtures/fixtures.py

```diff
@@ -58,7 +58,13 @@ def config_host(host):
 
 def connection_factory(request, user, host, ssh_priv_key):
     host, port = config_host(host)
-    connect_kwargs = {"password": "", "banner_timeout": 60, "auth_timeout": 60}
+    connect_kwargs = {
+        "password": "",
+        "banner_timeout": 60,
+        "auth_timeout": 60,
+        "look_for_keys": False,
+        "allow_agent": False,
```

Comment:
Does this prevent the password dialog during tests?

Reply:
Yes! :partying_face: 

I tested it locally and it works for me at least. We merged the same kind of fix on the integration test suite (see referred PRs above) and I haven't heard about tests not working locally for anybody.

I will do a full CI run before merge, though.

Reply:
Nice, very much a wanted feature!